handlers/services.go

```diff
@@ -104,13 +104,8 @@ func ServiceDetails(w http.ResponseWriter, r *http.Request) {
 			istioConfigValidations, errValidations = business.Validations.GetValidations(namespace, service)
 		}()
 	}
-	requestToken, err := getToken(r)
-	if err != nil {
-		RespondWithError(w, http.StatusInternalServerError, "Token initialization error: "+err.Error())
-		return
-	}
 
-	serviceDetails, err := business.Svc.GetService(namespace, service, rateInterval, queryTime, requestToken)
+	serviceDetails, err := business.Svc.GetService(namespace, service, rateInterval, queryTime)
```

Comment:
I don't understand why the `requestToken` has been removed. I think this is breaking an authentication case.

Reply:
Oh, I think I understand, the token is passed before, when creating the client, right?

Reply:
Yes The token is passed in the client